mungegithub/mungers/flakesync/cache.go

```diff
@@ -115,7 +121,9 @@ func NewCache(getFunc ResultFunc) *Cache {
 	c := &Cache{
 		byJob:             jobMap{},
 		flakeQueue:        flakeMap{},
+		expireList:        list.New(),
```

Comment:
oh, haha, this is probably worse. I was anchored because I thought of an LRU first. Doesn't matter much though.